openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts

```diff
@@ -421,7 +424,41 @@ test.describe('Teams Page', () => {
       'false'
     );
 
-    await clickOutside(page);
+    await visitOwnProfilePage(page);
+    await page.waitForLoadState('networkidle');
+
+    await page.getByTestId('edit-teams-button').click();
+    await page.getByTestId('team-select').click();
+
+    await expect(page.getByTestId('profile-teams-edit-popover')).toBeVisible();
+
+    await page
+      .getByTestId('profile-teams-edit-popover')
+      .getByText(publicTeam.displayName)
+      .click();
+
+    const updateUserResponse = page.waitForResponse('/api/v1/users/*');
+    await page.getByTestId('teams-edit-save-btn').click();
+    await updateUserResponse;
+
+    await expect(
+      page.getByTestId('profile-teams-edit-popover')
+    ).not.toBeVisible();
+
+    await expect(page.getByTestId('user-profile-teams')).toContainText(
+      publicTeam.displayName
+    );
+
+    await expect(page.getByTestId('user-profile-teams')).toContainText(
+      publicTeam.displayName
+    );
+
+    await page
+      .getByTestId('user-profile-teams')
+      .getByText(publicTeam.displayName)
+      .click();
```

Comment:
No need to have expect when eventually we are going to perform click on it. 
Directly perform click if it doesn't exists it will give us error